.github/workflows/main.yml

```diff
@@ -333,10 +333,9 @@ jobs:
         run: |
           $ch_prefix/from-tarball/bin/ch-test all
 
-      - name: ensure make uninstall works
+      - name: make uninstall
         if: ${{ matrix.builder == 'docker' && matrix.keep_sudo }}
         run: |
-          cd $ch_prefix/charliecloud-*
```

Comment:
I had to add this to get the CI to pass; the default working directory is not /var/tmp/from-tarball`.